config/config/sc-config.yaml

```diff
@@ -154,9 +171,21 @@ elasticsearch:
       limits:
         memory: 1024Mi
         cpu: 1
+    affinity:
+      podAntiAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+        - weight: 1
+          podAffinityTerm:
+            topologyKey: "kubernetes.io/hostname"
+            labelSelector:
+              matchLabels:
+                role: master
+    tolerations: []
+    nodeSelector: {}
   dataNode:
     count: 2
     storageSize: 25Gi
+    storageClass: local-storage
```

Comment:
Isn't this just for Exoscale?

Reply:
Yes, I don't know why we put this as default in the config but i guess it does not matter since it is always overwritten when running init https://github.com/elastisys/compliantkubernetes-apps/blob/f3c283316ecb5464932b1a25ff32116f9845eb86/bin/init.bash#L129-L133